README.md

```diff
@@ -11,13 +11,7 @@
 
 MaterialX is an open standard for representing rich material and look-development content in computer graphics, enabling its platform-independent description and exchange across applications and renderers.  Launched at [Industrial Light & Magic](https://www.ilm.com/) in 2012, MaterialX has been a key technology in their feature films and real-time experiences since _Star Wars: The Force Awakens_ and _Millennium Falcon: Smugglers Run_.  The project was released as open source in 2017, with companies including Sony Pictures Imageworks, Pixar, Autodesk, Adobe, and SideFX contributing to its ongoing development.  In 2021, MaterialX became the seventh hosted project of the [Academy Software Foundation](https://www.aswf.io/).
 
-## Quick Start for Developers
-
-- Download the latest version of the [CMake](https://cmake.org/) build system.
-- Point CMake to the root of the MaterialX library and generate C++ projects for your platform and compiler.
-- Select the `MATERIALX_BUILD_PYTHON` option to build Python bindings.
-- Select the `MATERIALX_BUILD_VIEWER` option to build the [MaterialX Viewer](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/documents/DeveloperGuide/Viewer.md).
-- Select the `MATERIALX_BUILD_GRAPH_EDITOR` option to build the [MaterialX Graph Editor](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/documents/DeveloperGuide/GraphEditor.md). 
+See the [Building Documentation](documents/DeveloperGuide/MainPage.md#building-materialx) for more information on how to build MaterialX.
```

Comment:
Is this change just left over from the first version?

Reply:
Oh, I see that it's pointing to the new location, which is good.  But perhaps there's still a value in maintaining this Quick Start Guide, and letting the new, improved link to the Developer Guide stand on its own?

Reply:
I added it back in, but added a few clarifying instructions for using CMake.

Reply:
I think your instinct is right that this section can be improved as well, but my sense is that we'll need more thought to get this exactly right.